ksqldb-functional-tests/src/test/resources/query-validation-tests/greatest.json

```diff
@@ -0,0 +1,139 @@
+{
+  "comments": [
+    "Tests covering the greatest function."
+  ],
+  "tests": [
+    {
+      "name": "test greatest with numeric types",
+      "format": ["JSON"],
```

Comment:
If the test is only using one format, then you can just inline the format instead of using the `format` property here. (Makes the test slightly more readable.)